packages/expo-updates/scripts/createManifest.js

```diff
@@ -1,3 +1,5 @@
+const filterPlatformAssetScales = require('@react-native-community/cli/build/commands/bundle/filterPlatformAssetScales')
```

Comment:
yikes, i could see this breaking easily. we should maybe follow up with an upstream pr to pull this into a separate package or a public api